workflows/argo/README.md

```diff
@@ -71,6 +71,8 @@ sequential segments.
 | `segment-count`      | (optional) Number of prognostic run segments; default "1"                     |
 | `cpu`                | (optional) Number of cpus to request; default "6"                             |
 | `memory`             | (optional) Amount of memory to request; default 6Gi                           |
+| `online-diags-flags` | (optional) `flags` for `prognostic-run-diags` workflow                        |
+| `online-diags`       | (optional) Run online diagostics if "true; default "true"                     | 
```

Comment:
```suggestion
| `online-diags`       | (optional) Run online diagostics if "true"; default "true"                     | 
```

Reply:
I didn't try introducing a fault in the code. I think it will work...